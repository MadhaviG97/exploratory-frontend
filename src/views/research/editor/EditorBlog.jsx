import React , { useEffect, useState } from 'react';
import classNames from "classnames";
import Footer from "../../../components/Footer/Footer";
import NavBar from "../../../components/Navbar/Navbar";
import axios from 'axios';

import Card from '@material-ui/core/Card';
import CardActionArea from '@material-ui/core/CardActionArea';
import CardContent from '@material-ui/core/CardContent';
import CardHeader from '@material-ui/core/CardHeader';
import CardActions from '@material-ui/core/CardActions';

import IconButton from '@material-ui/core/IconButton';
import DeleteIcon from '@material-ui/icons/Delete';
import ShareIcon from '@material-ui/icons/Share';
import MoreVertIcon from '@material-ui/icons/MoreVert';
import Divider from '@material-ui/core/Divider';
import Box from '@material-ui/core/Box';
import { useStyles } from "../../../assets/css/projectFolderGrid";
import Avatar from '@material-ui/core/Avatar';
import Grid from '@material-ui/core/Grid';




export default function CreatePage(props) {
 
    const [blogs, setBlogs] = useState([])
    const classes = useStyles();
    useEffect(() => {
        axios.post('/editor/getBlogs')
            .then(response => {
                if (response.data.success) {
                    console.log(response.data.blogs)
                    setBlogs(response.data.blogs)
                } else {
                    alert('Couldnt get blog`s lists')
                }
            })
    }, [])
    
    return(
        <div>
            <NavBar/>
            <Box p={2}></Box>
            <div className={classNames(classes.main, classes.mainRaised)}>

                <Box p={5}/> {/*marginTop={7} />*/}
                <div style={{ width: '85%', margin: '3rem auto' }}>
                    <div className={classes.name}  >
                        <h1 align='center' className={classes.title}>Group Name</h1>
                    </div>
                    <Box p={1} />
                    <Grid container spacing={4} direction="row" justify="center" alignItems="center">
                        {blogs.map((blog,index) => (
                        <Grid item lg={4} md={6} xs={12}>
                            <CardActionArea component="a" href={`/document/edit/${blog._id}`}>
                            <Card >
                                <CardHeader
                                    avatar={
                                        <Avatar aria-label="recipe" className={classes.avatar}>
                                          {blog.name[0]}
                                        </Avatar>
                                      }
                                    action={
                                    <IconButton aria-label="settings">
                                        <MoreVertIcon />
                                    </IconButton>
                                    }
                                    title={blog.name}
                                    subheader={blog.updatedAt}
                                />
                                <Divider variant="middle" />
                                <CardContent>
                                
                                    
                                    
                                    <div style={{ height: 150, overflowY: 'scroll', marginTop: 10 }}>
                                        <div dangerouslySetInnerHTML={{ __html: blog.content }} />
                                    </div>
                                    
                                </CardContent>
                                <Divider variant="middle" />
                                <CardActions disableSpacing>
                                    
                                    <IconButton aria-label="share">
                                    <ShareIcon />
                                    </IconButton>
                                    <IconButton aria-label="delete document"  >{/*href ={`/editor/delete/${blog._id}`} */}
                                    <DeleteIcon />
                                    </IconButton>
                                </CardActions>
                                
                                
                            </Card>
                            </CardActionArea>
                        </Grid>
                    ))}
                    </Grid>
                </div>
                
                
                <Box p={2}></Box>
            </div>
            
            <Footer/>
        </div>
    );

}