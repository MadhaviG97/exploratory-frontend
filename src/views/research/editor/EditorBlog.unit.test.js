import React from 'react';
import ReactDOM from 'react-dom';
import  EditorBlog  from './EditorBlog';
import { mount } from 'enzyme';
import axios from 'axios';
import { act } from 'react-dom/test-utils';
import configureMockStore from 'redux-mock-store'
import thunk from 'redux-thunk';
import {Provider} from 'react-redux'
import renderer from 'react-test-renderer';

const mockStore = configureMockStore([thunk]);
const mLocalStorage = {
  _storage: {},
  getItem: jest.fn((key) => {
    return mLocalStorage._storage[key];
  }),
  setItem: jest.fn((key, value) => {
    mLocalStorage._storage[key] = value;
  }),
};
Object.defineProperty(window, 'localStorage', {
  value: mLocalStorage,
});
const store = mockStore({
    user: { userData: {isAuth:true,first_name:'yogya'} }
  });
jest.mock('react-router-dom', () => ({
  useHistory: () => ({
    push: jest.fn(),
  }),
  useLocation: () => ({
    push: jest.fn(),
  }),
}));

describe('editor blog', () => {
  afterEach(() => {
    jest.restoreAllMocks();
    jest.resetAllMocks();
  });
  it('mocks API call', async () => {
    const token = 'JWT1111';
    mLocalStorage.setItem('token', token);
    const response = { data:{blogs: [ {name: 'mocked name',content:'<p>pol</p>'} ],success:true }};
    jest.spyOn(axios, 'post').mockResolvedValueOnce(response);
    const wrapper = mount(
        <Provider store={store}>
            <EditorBlog />
        </Provider>
        );
    await act(async () => {
      await new Promise((resolve) => setTimeout(resolve, 0));
    });
    //expect(wrapper.find('.org-docs-header').text()).toContain('mocked name');
    expect(axios.post).toBeCalledWith('/editor/getBlogs'/*, {
      headers: { Authorization: 'JWT ' + token },
    }*/);
  });
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
    <Provider store={store}>
        <EditorBlog />
    </Provider>, div);
  });
  it('renders correctly', () => {
      const tree = renderer
        .create(
        <Provider store={store}>
          <EditorBlog />
        </Provider>)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
});
