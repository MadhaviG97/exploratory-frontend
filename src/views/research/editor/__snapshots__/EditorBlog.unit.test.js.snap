// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`editor blog renders correctly 1`] = `
<div
  className="makeStyles-main2-13"
>
  <div>
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary makeStyles-appBar-57 mui-fixed MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular makeStyles-container-60 MuiToolbar-gutters"
      >
        <img
          alt="logo"
          src="/logo1.png"
        />
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-title-62"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Exploratory
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </button>
        <div
          className="makeStyles-flex-61"
        >
          <div
            className="PrivateHiddenCss-smDown-154"
          >
            <div>
              <ul
                className="MuiList-root makeStyles-list-173 MuiList-padding"
              >
                <li
                  className="MuiListItem-root makeStyles-listItem-174 MuiListItem-gutters"
                  disabled={false}
                >
                  <a
                    aria-disabled={false}
                    className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-205 makeStyles-transparent-220 makeStyles-navLink-176"
                    href="/"
                    onBlur={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={0}
                  >
                    <span
                      className="MuiButton-label"
                    >
                      Home
                    </span>
                    <span
                      className="MuiTouchRipple-root"
                    />
                  </a>
                </li>
                <li
                  className="MuiListItem-root makeStyles-listItem-174 MuiListItem-gutters"
                  disabled={false}
                >
                  <a
                    aria-disabled={false}
                    className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-205 makeStyles-transparent-220 makeStyles-navLink-176"
                    href="/aboutus"
                    onBlur={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={0}
                  >
                    <span
                      className="MuiButton-label"
                    >
                      About Us
                    </span>
                    <span
                      className="MuiTouchRipple-root"
                    />
                  </a>
                </li>
                <li
                  className="MuiListItem-root makeStyles-listItem-174 MuiListItem-gutters"
                  disabled={false}
                >
                  <a
                    aria-disabled={false}
                    className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-205 makeStyles-transparent-220 makeStyles-navLink-176"
                    href="/forum"
                    onBlur={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={0}
                  >
                    <span
                      className="MuiButton-label"
                    >
                      Forum
                    </span>
                    <span
                      className="MuiTouchRipple-root"
                    />
                  </a>
                </li>
                <li
                  className="MuiListItem-root makeStyles-listItem-174 MuiListItem-gutters"
                  disabled={false}
                >
                  <a
                    aria-disabled={false}
                    className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-205 makeStyles-transparent-220 makeStyles-navLink-176"
                    href="/project/createproject"
                    onBlur={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={0}
                  >
                    <span
                      className="MuiButton-label"
                    >
                      create project
                    </span>
                    <span
                      className="MuiTouchRipple-root"
                    />
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
        <div
          className="PrivateHiddenCss-smDown-154"
        >
          <ul
            className="MuiList-root makeStyles-list-237 MuiList-padding"
          >
            <li
              className="MuiListItem-root makeStyles-listItem-238 MuiListItem-gutters"
              disabled={false}
            >
              <div
                className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-adornedEnd MuiOutlinedInput-adornedEnd"
                onClick={[Function]}
              >
                <input
                  className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd MuiOutlinedInput-inputAdornedEnd"
                  id="outlined-adornment-password"
                  name="searchString"
                  onAnimationStart={[Function]}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  type="text"
                />
                <div
                  className="MuiInputAdornment-root MuiInputAdornment-positionEnd"
                >
                  <button
                    aria-label="search"
                    className="MuiButtonBase-root MuiIconButton-root MuiIconButton-edgeEnd"
                    disabled={false}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={0}
                    type="button"
                  >
                    <span
                      className="MuiIconButton-label"
                    >
                      <svg
                        aria-hidden="true"
                        className="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                        />
                      </svg>
                    </span>
                    <span
                      className="MuiTouchRipple-root"
                    />
                  </button>
                </div>
                <fieldset
                  aria-hidden={true}
                  className="PrivateNotchedOutline-root-312 MuiOutlinedInput-notchedOutline"
                  style={
                    Object {
                      "paddingLeft": 8,
                    }
                  }
                >
                  <legend
                    className="PrivateNotchedOutline-legend-313"
                    style={
                      Object {
                        "width": 0.01,
                      }
                    }
                  >
                    <span
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "&#8203;",
                        }
                      }
                    />
                  </legend>
                </fieldset>
              </div>
            </li>
            <li
              className="MuiListItem-root makeStyles-listItem-238 MuiListItem-gutters"
              disabled={false}
            >
              <div
                style={
                  Object {
                    "position": "relative",
                  }
                }
              >
                <button
                  className="MuiButtonBase-root MuiIconButton-root"
                  disabled={false}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={0}
                  type="button"
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <span
                      className="MuiBadge-root"
                    >
                      <svg
                        aria-hidden="true"
                        className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 22c1.1 0 2-.9 2-2h-4c0 1.1.89 2 2 2zm6-6v-5c0-3.07-1.64-5.64-4.5-6.32V4c0-.83-.67-1.5-1.5-1.5s-1.5.67-1.5 1.5v.68C7.63 5.36 6 7.92 6 11v5l-2 2v1h16v-1l-2-2z"
                        />
                      </svg>
                      <span
                        className="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorPrimary"
                      >
                        4
                      </span>
                    </span>
                  </span>
                  <span
                    className="MuiTouchRipple-root"
                  />
                </button>
              </div>
              <div
                style={
                  Object {
                    "bottom": 20,
                    "minWidth": 325,
                    "position": "fixed",
                    "right": 25,
                    "zIndex": 1,
                  }
                }
              />
            </li>
            <li
              className="MuiListItem-root makeStyles-listItem-238 MuiListItem-gutters"
              disabled={false}
            >
              <div>
                <div>
                  <button
                    aria-haspopup="true"
                    aria-label="Notifications"
                    aria-owns={null}
                    className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-205 makeStyles-transparent-220 makeStyles-navLink-240 makeStyles-imageDropdownButton-253"
                    disabled={false}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={0}
                    type="button"
                  >
                    <span
                      className="MuiButton-label"
                    >
                      <img
                        alt="profile"
                        className="makeStyles-img-252"
                        src="data:image/jpeg;base64,undefined"
                      />
                      <div
                        className="makeStyles-imageLink-242"
                        color="transparent"
                      >
                        yogya
                      </div>
                    </span>
                    <span
                      className="MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
            </li>
          </ul>
        </div>
      </div>
    </header>
  </div>
  <div
    className="MuiCollapse-container MuiCollapse-hidden"
    style={
      Object {
        "minHeight": "0px",
      }
    }
  >
    <div
      className="MuiCollapse-wrapper"
    >
      <div
        className="MuiCollapse-wrapperInner"
      >
        <div
          className="MuiPaper-root MuiAlert-root MuiAlert-standardSuccess MuiPaper-elevation0"
          role="alert"
        >
          <div
            className="MuiAlert-icon"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M20,12A8,8 0 0,1 12,20A8,8 0 0,1 4,12A8,8 0 0,1 12,4C12.76,4 13.5,4.11 14.2, 4.31L15.77,2.74C14.61,2.26 13.34,2 12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0, 0 22,12M7.91,10.08L6.5,11.5L11,16L21,6L19.59,4.58L11,13.17L7.91,10.08Z"
              />
            </svg>
          </div>
          <div
            className="MuiAlert-message"
          >
            Document Succesfully Deleted!
          </div>
          <div
            className="MuiAlert-action"
          >
            <button
              aria-label="close"
              className="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeSmall"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                  />
                </svg>
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div
      className="MuiBox-root MuiBox-root-601"
    />
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-5"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3"
      >
        <div
          className="MuiBox-root MuiBox-root-602"
        />
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            className="MuiBox-root MuiBox-root-603"
          >
            <div>
              <ul
                className="MuiList-root makeStyles-root-479 MuiList-padding"
              >
                <li
                  className="MuiListItem-root MuiListItem-gutters MuiListItem-alignItemsFlexStart"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      <form
                        className="MuiPaper-root makeStyles-root-479 MuiPaper-elevation1 MuiPaper-rounded"
                      >
                        <div
                          className="MuiInputBase-root makeStyles-input-481"
                          onClick={[Function]}
                        >
                          <input
                            aria-label="search drive"
                            className="MuiInputBase-input"
                            onAnimationStart={[Function]}
                            onBlur={[Function]}
                            onChange={[Function]}
                            onFocus={[Function]}
                            placeholder="Search Documents"
                            type="text"
                          />
                        </div>
                        <button
                          aria-label="search"
                          className="MuiButtonBase-root MuiIconButton-root makeStyles-iconButton-482"
                          disabled={false}
                          onBlur={[Function]}
                          onClick={[Function]}
                          onDragLeave={[Function]}
                          onFocus={[Function]}
                          onKeyDown={[Function]}
                          onKeyUp={[Function]}
                          onMouseDown={[Function]}
                          onMouseLeave={[Function]}
                          onMouseUp={[Function]}
                          onTouchEnd={[Function]}
                          onTouchMove={[Function]}
                          onTouchStart={[Function]}
                          tabIndex={0}
                          type="submit"
                        >
                          <span
                            className="MuiIconButton-label"
                          >
                            <svg
                              aria-hidden="true"
                              className="MuiSvgIcon-root"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                              />
                            </svg>
                          </span>
                          <span
                            className="MuiTouchRipple-root"
                          />
                        </button>
                      </form>
                    </span>
                  </div>
                </li>
                <hr
                  className="MuiDivider-root"
                />
                <li
                  className="MuiListItem-root MuiListItem-gutters MuiListItem-alignItemsFlexStart"
                  disabled={false}
                >
                  <div
                    className="MuiListItemAvatar-root MuiListItemAvatar-alignItemsFlexStart"
                  >
                    <div
                      className="MuiAvatar-root MuiAvatar-square"
                    >
                      <img
                        alt="Cindy Baker"
                        className="MuiAvatar-img"
                        src="undefined/images/fileFolder/fileAvatar.png"
                      />
                    </div>
                  </div>
                  <div
                    className="MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      <label
                        aria-disabled={false}
                        backgroundcolor="#b2beb5"
                        className="MuiButtonBase-root MuiButton-root MuiButton-text"
                        height={50}
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex={0}
                      >
                        <span
                          className="MuiButton-label"
                        >
                          <span
                            className="MuiTypography-root makeStyles-inline-480 MuiTypography-body2 MuiTypography-colorTextPrimary"
                          >
                            Create New
                          </span>
                        </span>
                        <span
                          className="MuiTouchRipple-root"
                        />
                      </label>
                    </span>
                  </div>
                </li>
                <hr
                  className="MuiDivider-root"
                />
                <li
                  className="MuiListItem-root MuiListItem-gutters MuiListItem-alignItemsFlexStart"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      <label
                        aria-disabled={false}
                        backgroundcolor="#b2beb5"
                        className="MuiButtonBase-root MuiButton-root MuiButton-text"
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex={0}
                      >
                        <span
                          className="MuiButton-label"
                        >
                          <span
                            className="MuiTypography-root makeStyles-inline-480 MuiTypography-body2 MuiTypography-colorTextPrimary"
                          >
                            Compare Two Documents
                          </span>
                        </span>
                        <span
                          className="MuiTouchRipple-root"
                        />
                      </label>
                    </span>
                  </div>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
      <hr
        className="MuiDivider-root MuiDivider-vertical"
      />
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
      >
        <div
          className="MuiBox-root MuiBox-root-604"
          style={
            Object {
              "display": "flex",
            }
          }
        >
          <div
            className="MuiBox-root MuiBox-root-605"
          >
            <div>
              <button
                aria-describedby={null}
                className="MuiButtonBase-root MuiIconButton-root"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseOver={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
                title="Apps"
                type="button"
              >
                <span
                  className="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M4 8h4V4H4v4zm6 12h4v-4h-4v4zm-6 0h4v-4H4v4zm0-6h4v-4H4v4zm6 0h4v-4h-4v4zm6-10v4h4V4h-4zm-6 4h4V4h-4v4zm6 6h4v-4h-4v4zm0 6h4v-4h-4v4z"
                    />
                  </svg>
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
        <hr
          className="MuiDivider-root"
        />
        <div
          className="MuiBox-root MuiBox-root-606"
        />
        <div
          className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4"
        />
      </div>
    </div>
  </div>
  <div
    className="MuiBox-root MuiBox-root-607"
  />
</div>
`;
